examples/foundry/test/TestPrecompiles.t.sol

```diff
@@ -0,0 +1,174 @@
+// SPDX-License-Identifier: MIT
```

Comment:
These tests should probably be moved and executed inside the integrations/ package and not the example.

Reply:
Added separate tests for integrations/foundry and examples/foundry